t/03-xsstatic.t

```diff
@@ -11,9 +11,6 @@ use MakeMaker::Test::Utils;
 use MakeMaker::Test::Setup::XS;
 use Test::More;
 
-plan skip_all => "Disabled as broken perl installs give false negative"
-  # if not static perl, and not author
-  unless !$Config{usedl} or $ENV{AUTHOR_TESTING};
```

Comment:
I recall a great deal of discussion that ended with default-disabling the static-build tests with this. What's the rationale behind default-enabling?